packages/data/addon/models/metadata/metric/function-argument.js

```diff
@@ -59,14 +64,12 @@ export default class FunctionArgument extends EmberObject {
    */
   get values() {
     if (this.expression === 'self') {
-      return new Promise(resolve => {
-        resolve(this._localValues);
-      });
+      return Promise.resolve(this._localValues);
     }
 
     const [type, id] = this.expression.split(':');
     if (this.type === 'ref' && type && id) {
-      return this.metadataService.findById(type, id);
+      return this.metadataService.findById(type, id), this.source;
```

Comment:
Sneaky comma operator
```suggestion
      return this.metadataService.findById(type, id, this.source);
```